all-products.html

```diff
@@ -11,59 +11,97 @@
 		</header>
 		
 		<div class="wrapper">
-			<!-- This div is for the products on the page-->
 			<div class="products">
                 <h1>All Products</h1>
 
-                <!-- Flexbox to divide each row easier-->
-                <div class="flex-container">
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
-                    <div class="flex-child"><p>Image will go here</p></div>
+                <div class="container">

```

Comment:
I used layout 2's overall layout for layout 3's "All Products" page, and the comparison can be seen in Pull request #1.